### PR TITLE
Fix results

### PR DESCRIPTION
**Pull request by dvalinrh**

### CODE REVIEW

File: uperf/uperf_run [hunk 1]
```diff
@@ -114,7 +114,7 @@ usage()
 	echo "  --packet_type:  comma separated list of packet types, (tcp,udp)"
 	echo "  --server_ips: comma separated list of the server ip addresses"
 	echo "  --suffix: Suffix to add to the results file"
-	echo "  --test_types:  stream,rr,maerts,bidirec"
+	echo "  --tests:  stream,rr,maerts,bidirec"
```
Comment (kdvalin) [inline uperf/uperf_run:117]: Is this necessary for the results fix?

**Review by kdvalin — CHANGES_REQUESTED** [review]
Only comment is renaming `--test_types` to `--tests`

We should be avoiding that if possible, if you want to add an alias for it in a separate PR go ahead, but most people would probably forget about it in their setup and then reach headaches.

File: uperf/uperf_run (hunk 1 above)
Comment (dvalinrh) [reply uperf/uperf_run:117]: It is just changing the usage to what it should be.  If you want to be nitty we can open a separate pr and make the the change there.

Comment (frival) [reply uperf/uperf_run:117]: As long as it's separate commits I'm not going to jump off a cliff over this one.  Just one rap on the knuckles with a ruler this time.

Comment (dvalinrh) [reply uperf/uperf_run]: This has been moved to its own pr.

File: uperf/uperf_run [hunk 2]
```diff
@@ -596,15 +596,20 @@ run_pbench_uperf_test()
 #
 save_pbench_results()
 {
+	test_results_file_out="Failed"
 	cd /var/lib/pbench-agent/
-	mkdir export_uperf_data
+	mkdir $uperf_results
```
Comment (kdvalin) [inline uperf/uperf_run:601]:
```suggestion
```

Can be condensed down, needs to go with the next suggestion.

File: uperf/uperf_run [hunk 3]
```diff
@@ -596,15 +596,20 @@ run_pbench_uperf_test()
...
+	test_results_file_out="Failed"
 	cd /var/lib/pbench-agent/
-	mkdir export_uperf_data
+	mkdir $uperf_results 
 	for i in `ls -d uperf_*`
 	do
-		mkdir export_uperf_data/$i
-		cp $i/result.csv export_uperf_data/$i
-		cp $i/result.txt export_uperf_data/$i
+		mkdir $uperf_results/$i
```
Comment (kdvalin) [inline uperf/uperf_run:604]:
```suggestion
		mkdir -p $uperf_results/$i
```

**Review by kdvalin — APPROVED** [review]
Overall, LGTM, you can remove a redundant mkdir line, but it's not critical

**Review by kdvalin — APPROVED** [review]
LGTM

**Pull request merged**